docs/Authenticating-Users-via-SAML.md

```diff
@@ -216,7 +211,14 @@ Rebuild the WAR file and re-deploy:
 
 ```
 mvn -DskipTests clean install
-cp portal/target/cbioportal.war $CATALINA_HOME/webapps/
+java \
+    -Dauthorization=true \
+    -Dauthenticate=saml \
+    -Ddbconnector=dbcp \
```

Comment:
@inodb what are the plans to get these properties from `portal.properties` instead? I feel we should actually try to get rid of all -D options in the example commands. IMO the documentation should tell users to adjust properties in the .properties file. We could then have only one section in documentation (for "advanced" users) telling how to overwrite portal.properties on runtime.

Reply:
@pieterlukasse that makes sense - I think all of these can go in `portal.properties` (those can btw also be overridden on runtime using `PORTAL_HOME` env variable). There was one issue with the `authorization` variable I think which required it to be set with `-D`. That's how it ended up in the documentation before (using `CATALINA_OPTS`). Maybe @fedde-s can elaborate what the issue was there

Reply:
True; for the `authenticate` variable, cBioPortal cannot currently use portal.properties at runtime. When implementing runtime configuration of auth in PR https://github.com/cBioPortal/cbioportal/pull/4917, none of us knew how to set up the Spring configuration (profile selection in XML files) to support that. That should be tracked as a separate issue, not introduced by this PR here.

For all other properties, I agree that we should document them in a consistent way, in portal.properties. Highlighting the specific settings to change for the use-case documented on this page from the ‘boilerplate code’ of the command would also address the issue I pointed out in my other review comment.

Reply:
@pieterlukasse @fedde-s agree with these comments. Let's only use `portal.properties`. With the exception for `authenticate`. Created a separate issue for that: https://github.com/cBioPortal/cbioportal/issues/6109.

As a side note, personally I prefer command line arguments over config files if there are proper `--help` / `man` docs, but I might be in the minority there. Regardless we are v far away from that, so using `properties` file def seems more feasible.